sdks/python/apache_beam/transforms/core_test.py

```diff
@@ -0,0 +1,63 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I think it's worth adding a note to CHANGES.md in Breaking Changes section, with a suggestion how users should proceed if they are affected.

Reply:
will do.

Reply:
Please take a look at CHANGES.md.

Reply:
No need to change this now.